Configure

```diff
@@ -561,6 +561,9 @@ $config{build_type} = "release";
 
 my %unsupported_options = ();
 my %deprecated_options = ();
+# If you change this, update apps/version.c
+my @known_seed_sources = qw(getrandom devrandom os egd none);
```

Comment:
Does this mean that you should be able to configure, say `--with-rand-seed=getrandom,os`?  Or `--with-rand-seed=getrandom,none`?  Or `--with-rand-seed=os,none`?